### DISCUSSION TITLE
Is a new release planned for contribution to SimRel 2023-09?

### DISCUSSION THREAD

**Discussion by merks** [eclipse/tm4e#588] [Q&A] [answered]
Looking at https://download.eclipse.org/tm4e/snapshots/ it has newer dependencies and it would be nice to get rid of old duplicate versions of Guava:

![image](https://github.com/eclipse/tm4e/assets/208716/d8735e0d-d038-443c-b856-87dac02af4aa)

Is a new release planned?

**Comment by mickaelistria** [comment, marked as answer]
OK, I'm releasing 0.8.1.

**Reply by merks** [reply]
Ping me here and I will look for https://download.eclipse.org/tm4e/releases/0.8.1 and update the contribution for you if you wish.

Thanks!!
